datadog/tests/resource_datadog_security_monitoring_default_rule_test.go

```diff
@@ -43,26 +45,28 @@ func idFromDatasource(state *terraform.State) (string, error) {
 	return resourceState.Primary.Attributes["rule_ids.0"], nil
 }
 
-func testAccCheckDatadogSecurityMonitoringDefaultDatasource() string {
+func testAccDatadogSecurityMonitoringDefaultDatasource() string {
 	return `
 data "datadog_security_monitoring_rules" "bruteforce" {
-    name_filter = "docker"
+	tags_filter = ["source:cloudtrail"]
```

Comment:
This makes sure we're importing a default rule of type log detection. CloudTrail rules should all be of type log detection. The previous filter returned an infrastructure configuration rule.